frontend/src/actions/courseActions.js

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+export const courseActions = (action, payload) => {
+    switch (action) {
+        case 'APPEND':
+            return { 
+                type: 'APPEND', 
+                payload: payload
+            }
+        case 'DELETE':
+            return { 
+                type: 'DELETE', 
+                payload: payload, 
+            }
+        case 'SET_COURSES': 
+            return {
+                type: 'SET_COURSES',
+                payload: payload
+            }
+        case 'SET_CORE_COURSES': 
+            console.log('SETTING CORE', payload)
+            return {
+                type: 'SET_CORE_COURSES',
+                payload: payload
+            }
+        // case 'GET_COURSE_BY_ID':
```

Comment:
@gabriellahartanto  This should be fetched from backend. See `courseProvider.js`